registration-processor/core-processor/registration-processor-manual-adjudication-stage/src/main/java/io/mosip/registration/processor/adjudication/service/impl/ManualAdjudicationServiceImpl.java

```diff
@@ -1012,24 +1013,40 @@ private void checkAndUpdateIdentity(ManualVerificationEntity entity,
 				uins.add(uin);
 			}
 		}
-
+		String handleId ="";
+		List<String> matchedCurpIds = new ArrayList<>();
 		regProcLogger.info("ManualAdjudication::checkAndUpdateIdentity, uins: {}", uins.toString());
 		if (!uins.isEmpty() && uins.size() == 1) {
-			IdRequestDto idRequestDto = prepareIdRepoRequest(entity.getRegId(), uins.stream().findFirst().get(), registrationStatusDto.getRegistrationType());
+			handleId = packetManagerService.getFieldByMappingJsonKey(entity.getRegId(), MappingJsonConstants.CURPID,
+					registrationStatusDto.getRegistrationType(), ProviderStageName.MANUAL_ADJUDICATION);
+			matchedCurpIds = fetchHandlesToUpdate(entity.getRegId(), uins.stream().findFirst().get(), registrationStatusDto.getRegistrationType());
+			IdRequestDto idRequestDto = prepareIdRepoRequest(matchedCurpIds,entity.getRegId(), uins.stream().findFirst().get(), registrationStatusDto.getRegistrationType());
 			regProcLogger.info("ManualAdjudication::checkAndUpdateIdentity, Update Identity Request: {}", mapper.writeValueAsString(idRequestDto));
 			ResponseDTO responseDTO = idRepoService.updateIdentity(idRequestDto);
 			regProcLogger.info("ManualAdjudication::checkAndUpdateIdentity, Update Identity Response: {}", mapper.writeValueAsString(responseDTO));
 		}
 //		- update the curp_bio_data with status 'DUPLICATE'
 //		- insert into matched_curp table
+		CurpBioDtorequest curpBioDtorequest = new CurpBioDtorequest();
+		curpBioDtorequest.setCurpId(handleId);
+		curpBioDtorequest.setMatchedCurpIds(matchedCurpIds);
+		curpBioDtorequest.setStatusCode("ACTIVATE");
+		curpBioDtorequest.setStatusComment("Curp Id Activated");
+		curpBioDtorequest.setCurpStatus("DUPLICATE");
+		try {
+			regProcLogger.debug("Sending POST request to API: " + ApiName.CRUPMANAGERAPI + " with request body: " + curpBioDtorequest);
+			String response = (String) registrationProcessorRestClientService.postApi(ApiName.CRUPMANAGERAPI, new ArrayList<>(), null, null, curpBioDtorequest, String.class);

```

Comment:
Proper url for "CRUPMANAGERAPI" keyword should be defined in mosip-config under registration-processor-default.properties file.